omnigibson/robots/tiago.py

```diff
@@ -572,11 +572,7 @@ def get_position_orientation(self):
 
     def set_position_orientation(self, position=None, orientation=None):
         current_position, current_orientation = self.get_position_orientation()
-        if position is None:
-            position = current_position
-        if orientation is None:
-            orientation = current_orientation
```

Comment:
we probably shouldn't remove these lines?

Reply:
Sorry, this was an accident. I added these lines back in and re-pushed.